llama_stack/providers/adapters/inference/together/config.py

```diff
@@ -4,17 +4,12 @@
 # This source code is licensed under the terms described in the LICENSE file in
 # the root directory of this source tree.
 
-from pydantic import BaseModel, Field
-
 from llama_models.schema_utils import json_schema_type
-
-from llama_stack.distribution.request_headers import annotate_header
+from pydantic import BaseModel, Field
 
 
 class TogetherHeaderExtractor(BaseModel):
```

Comment:
is this class needed? I think you only need the ProviderDataValidator class right?